src/test/java/org/ahocorasick/trie/PayloadTrieTest.java

```diff
@@ -449,8 +449,9 @@ public void testLargeString() {
         assertEquals(textSize / interval, emits.size());
     }
 
+    // @see https://github.com/robert-bor/aho-corasick/issues/85
```

Comment:
See note about external links to GitHub.

Reply:
where is the note? I saw it's pretty common in this unit test class to add links to the related issues. I think it's a good for reference.